spec/spec_helper_acceptance.rb

```diff
@@ -45,13 +45,13 @@ def expect_no_change_from_manifest(manifest)
 end
 
 def http_package_url
-  url_root = 'http://download.elasticsearch.org/logstash/logstash/packages'
+  url_root = 'https://download.elastic.co/logstash/logstash/packages'
 
   case fact('osfamily')
   when 'Debian'
-    "#{url_root}/debian/logstash_#{LS_VERSION}-1_all.deb"
+    "#{url_root}/debian/logstash-#{LS_VERSION}_all.deb"
```

Comment:
The `-1` naming convention appears to have been dropped for Logstash 2.4.0, so this change works for that version, but I think it also explains why you are having trouble with earlier versions, like 2.3.4. By way of example, here are the download URLs for 2.3.4 and 2.4.0.

`https://download.elastic.co/logstash/logstash/packages/debian/logstash_2.3.4-1_all.deb`
`https://download.elastic.co/logstash/logstash/packages/debian/logstash-2.4.0_all.deb`